SIT/environment.sh

```diff
@@ -81,7 +81,7 @@ _testDockerNetwork() {
 }
 
 _testDockerCassandra() {
-  dockerPs=$(docker ps -a | awk '{if ($NF == "'${DOCKER_CASS}'") {print "yes"}}')
+  dockerPs=$(docker ps | awk '{if ($NF == "'${DOCKER_CASS}'") {print "yes"}}')
```

Comment:
Now with this change in place, I'm running into this error,
```
INFO Starting Docker container cdm-sit-cass
docker: Error response from daemon: Conflict. The container name "/cdm-sit-cass" is already in use by container "37ef23ba475bc76e1e80fec2b8c80edbc336a5c1149261912fbfd2571f1b0ad0". You have to remove (or rename) that container to be able to reuse that name.
See 'docker run --help'.
```

Could you please showcase how you originally encountered the error that you have in the PR description and provide reproduction steps? TY!

Reply:
Try this - kill your ITs in the middle of the run after the docker containers come up and then restart your docker engine. This will stop the running containers, but they will remain there and continue to get returned in the `docker ps -a` output.